cmd/migrate.go

```diff
@@ -99,6 +103,15 @@ func migrate(cmd *cobra.Command, args []string) {
 	shipment := args[0]
 	env := args[1]
 
+	//validate that the "app-env" name (used for alb name) is <= 32 characters
+	app := shipment
+	if migrateAppName != "" {
+		app = migrateAppName
+	}
+	if len(app+env) > 32 {
```

Comment:
if len(app+env)+1 > 32 {

+1 to account for the "-" between app and env.

Reply:
Nice catch!